run.sh

```diff
@@ -346,6 +349,25 @@ check_plugins() {
   done <<< "$(WP plugin list --field=name)"
 }
 
+check_packages() {
+  # If a composer.json file exists in /app => install it
+  if [[ -f "/app/composer.json" ]]; then
+    h3 "Installing packages listed in composer.json"
+    composer install
+    return
+  fi
+
+  if [[ "${REQUIRE-}" ]]; then
```

Comment:
This doesn't install from both, but gives both options to developers: they can start by adding their packages to their docker-compose.yml, and later move it to a separate composer.json file when they feel this clutters their ENV too much. Using REQUIRE will be easier for beginners (me, for example), while using a separate composer.json will be easier for people used to Composer.